build.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 # We just wrap `build` so this is really it
 name: "jellyfin-ffmpeg"
-version: "5.1.2-8"
+version: "6.0-1"
```

Comment:
```suggestion
version: "6.0.0-1"
```

Reply:
How this affects our workflow? I thought 6.0 and 6.0.0 are equal.

Reply:
I'm not sure if Debian likes it if you swith between `Major.Minor` and `Major.Minor.Patch`

IMHO we should stay consistent and use `Major.Minor.Patch` here and in the package version.

Reply:
For completeness: Debian has no problem with this, so ignore the comment.